ui/src/router/components/form/validation/schema.js

```diff
@@ -92,7 +92,9 @@ const routerNameRegex = /^[a-z0-9-]*$/,
   kafkaBrokersRegex =
     /^([a-z]+:\/\/)?\[?([0-9a-zA-Z\-%._:]*)\]?:([0-9]+)(,([a-z]+:\/\/)?\[?([0-9a-zA-Z\-%._:]*)\]?:([0-9]+))*$/i,
   kafkaTopicRegex = /^[A-Za-z0-9_.-]{1,249}/i,
-  trafficRuleNameRegex = /^[A-Za-z\d][\w\d \-()#$%&:.]*[\w\d\-()#$%&:.]$/;
+  trafficRuleNameRegex = /^[A-Za-z\d][\w\d \-()#$%&:.]*[\w\d\-()#$%&:.]$/,
+  upiUrlRegex= /^([a-zA-Z0-9_-]{1,256})(\.[a-zA-Z0-9-]{1,253})*(:\d+)$/,
+  serviceMethodRegex= /^\/?(\w+(\.|-)?)+(\/[A-Za-z]+)$/
```

Comment:
![Uploading image.png…]()

Reply:
Why the last one didn't match? Is numeric not allowed in the method name?

Reply:
yup numeric isn't allowed in method name

Reply:
I just found out that grpc allows method name containing numeric value.

Reply:
added numeric to regex